caproto/_broadcaster.py

```diff
@@ -70,11 +83,15 @@ def send(self, *commands):
             bytes to send over a socket
         """
         bytes_to_send = b''
-        self.log.debug("Serializing %d commands into one datagram",
-                       len(commands))
         history = []
+        total_commands = len(commands)
         for i, command in enumerate(commands):
-            self.log.debug("%d of %d %r", 1 + i, len(commands), command)
+            tags = {'role': repr(self.our_role)}
+            if hasattr(command, 'name'):
+                tags['pv'] = command.name
+            for address in self.our_addresses:
+                tags['our_address'] = address
+            search_logger.debug(f"%d of %d %r", 1 + i, total_commands, command, extra=tags)
```

Comment:
This line needs to be indented so that each address gets logged. As written, it will loop through `self.our_addresses` and then only log the _last_ address in the loop.

Reply:
Is it possible that some command need to be logged while self._our_address(I think it should be self._our_address, typo self.our_address) is still empty. Then we skip those command, ex beacon.